handbook/communication.md

```diff
@@ -324,10 +324,12 @@ There are times when presenting during a meeting is needed. This may occur when
 
 ## Indicating availability
 
-Indicate your availability by updating your own calendar using Google's ["out of office"](https://www.theverge.com/2018/6/27/17510656/google-calendar-out-of-office-option) feature and include the dates you plan to be away in your automated response. Note that this feature will automatically decline any meeting invitations during the time frame you select.
+For planned away time during working hours (e.g. holidays, vacation, travel time, and other leave), indicate your availability via the all three of the following:
+1. Update your own calendar using Google's ["out of office"](https://www.theverge.com/2018/6/27/17510656/google-calendar-out-of-office-option) feature. Note that this feature will automatically decline any meeting invitations during the time frame you select.
+  * Additionally, add any planned vacation time to the Analytics Team Calendar
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏@Rosi-D I just included the Analytics Team Calendar. Should we instruct the team to update the Prod & Tech Vacation calendar as well?

Reply:
@waligob Any planned OOO time should go on to the prod tech calendar.  The prod tech calendar is the universal place for folks to put scheduled OOO time.  Toan invited everyone to this calendar during the DE team agreement meeting so everyone should have access.

Reply:
Thanks for the response, @Rosi-D. I updated the details in https://github.com/phillymedia/data-engineering-handbook/pull/32/commits/e09aebdb67553884ae57ce97bb72665cbcc10080